src/commands/env.rs

```diff
@@ -7,7 +7,9 @@ use crate::path_ext::PathExt;
 use crate::shell::{infer_shell, Shell, AVAILABLE_SHELLS};
 use colored::Colorize;
 use std::fmt::Debug;
+use std::process::Command as SystemCommand;
```

Comment:
why do we need to import `Command`? can we use `use std::process` and `process::Command`?